modules/olm-restricted-networks-configuring-operatorhub.adoc

```diff
@@ -48,12 +50,15 @@ mirroring ...
 <1> Specify your Operator catalog image.
 <2> Specify a namespace to mirror all referenced images.
 +
-This generates an ImageContentSourcePolicy object that can configure nodes to
-translate between the image references stored in Operator manifests and the
-mirrored registry. A `mapping.txt` file is also created that which contains all
-of the source images and where to map them to in the target registry. This file
-is compatible with `oc image mirror` and can be used to further customize the
-mirroring configuration.
+This command also generates the following manifests:
++
+--
+* An ImageContentSourcePolicy object that can configure nodes to translate between
```

Comment:
s/ImageContentSourcePolicy/`ImageContentSourcePolicy` 
Seems to be the preferred way to present this object.